docs/en/Running-Hadoop-MapReduce-on-Alluxio.md

```diff
@@ -148,3 +148,7 @@ in Alluxio. You can see the resulting files by running:
 $ bin/alluxio fs ls /wordcount/output
 $ bin/alluxio fs cat /wordcount/output/part-r-00000
 ```
+
+> Tips：The previous wordcount example is also applicable to Alluxio in fault tolerant mode with Zookeeper. 
```

Comment:
link to detailed docs so that users can decide for themselves whether to use config vs uri